pubspec.yaml

```diff
@@ -10,9 +10,9 @@ environment:
   sdk: '>=2.15.0 <3.0.0'
 
 dependencies:
-  archive: ^3.1.2
-  meta: ^1.3.0
-  xml: ^6.0.1
+  archive: '>=3.1.1 <4.0.0'
+  meta: '>=1.3.0 <2.0.0'
```

Comment:
But `^1.3.0` and `>=1.3.0 <2.0.0` means the same thing in pubspec.yaml.

Reply:
Oh, yeah you're right. It confuses me because of how often i need to override my deps :roll_eyes: i guess the community just keeps bumping the last patch number too, and that breaks stuff